src/types/generated/types/ObjectParamAPI.d.ts

```diff
@@ -4863,6 +4863,12 @@ export interface OrgSettingApiUpdateOrgContactUserRequest {
     orgContactUser: OrgContactUser;
 }
 export interface OrgSettingApiUpdateOrgLogoRequest {
+    /**
+      *
+      * @type HttpFile
+      * @memberof OrgSettingApiupdateOrgLogo
+      */
+    file: HttpFile;
```

Comment:
it looks like we need to update HttpFile type to indicate this is a stream rather than {data, name} object.